instrumentation-genai/opentelemetry-instrumentation-vertexai/src/opentelemetry/instrumentation/vertexai/utils.py

```diff
@@ -137,6 +150,21 @@ def get_genai_request_attributes(
     return attributes
 
 
+def get_genai_response_attributes(
+    response: prediction_service.GenerateContentResponse
+    | prediction_service_v1beta1.GenerateContentResponse,
+) -> dict[str, AttributeValue]:
+    finish_reasons: list[str] = [
+        candidate.finish_reason.name for candidate in response.candidates
+    ]
+    return {
```

Comment:
Could we add `gen_ai.response.id` as well? From the [doc](https://cloud.google.com/vertex-ai/docs/reference/rest/v1/GenerateContentResponse) it appears that it's available in the response.

Reply:
Good find. Unfortunately it's not available in the python client today so will need to follow up on that